modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/FieldBase/ReactFieldBase.es.js

```diff
@@ -233,6 +233,57 @@ function FieldBase({
 			type === 'radio');
 	const showPopover = fieldName === 'inputMaskFormat';
 
+	useLayoutEffect(() => {
+		focusRequiredElement();
+	});
```

Comment:
@joaovictoralvesmartins, I just realized that we should add the array of dependencies, like below:
```
useLayoutEffect(() => {
    focusRequiredElement();
}, []);
```
Can you please add it?

In addition, do you have any update regarding this [comment](https://github.com/liferay-forms/liferay-portal/pull/724#discussion_r665625504)?

Reply:
@joaovictoralvesmartins, I think I've just found the reason why this bug doesn't reproduce on Text and Numeric fields. :slightly_smiling_face: 

Please check this [code](https://github.com/liferay/liferay-portal/blob/master/modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/core/reducers/pageValidationReducer.es.js#L26-L77).

Reply:
@natocesarrego, i did look at the code you provided and i found out that this behavior happens with other fields because for some reason, it uses hidden input fields in the form, and the reason why it's not focusing correctly, is because the code tries to focus exactly on a hidden input, which you can't do, unless you make some kind of workaround. Any ideas?